public/src/components/channelManagement/campaigns/CampaignCopyButton.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import useOpenable from '../../../hooks/useOpenable';
+import { Button, makeStyles, Theme, Typography } from '@material-ui/core';
```

Comment:
This PR skips the opportunity to display the `copy` button in the sticky top bar. The functionality of such a button is going to be much more involved than copying a test as we need to make decisions about whether copying a campaign also automatically copies all the tests associated with that campaign.

I think solving the copy dilemma is of lower priority compared to the functionality we want to add to the campaign display - especially listing all the tests associated with a campaign (including links to the individual test views)